keycloak-operator/values.yaml

```diff
@@ -0,0 +1,396 @@
+###############################################################################
+
+#########################
+# Global Configurations #
+#########################
+
+global:
+  # Overrides image.registry
+  # Can be used to override it globally when using umbrella charts.
+  imageRegistry: ""
+
+  # Inserts additional pull secret references together along with any from .Values.image.pullSecrets
+  # Can be used to override it globally when using umbrella charts.
+  # Values are processed as Helm templates.
+  # Ref: https://kubernetes.io/docs/concepts/configuration/secret/#using-imagepullsecrets
+  #
+  # Example:
+  # imagePullSecrets:
+  #   - name: my-pull-secret
+  imagePullSecrets: []
+
+  # The default domain suffix of the Kubernetes cluster
+  # Ref: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/
+  clusterDomain: cluster.local
+
+###########################
+# Metadata Configurations #
+###########################
+
+# Overrides the name of the chart affecting the names of the resources.
+nameOverride: ""
+
+# Overrides the naming of all resources, effectively removing the chart's name and the release name prefix.
+# This override takes precedence over anything in .Values.nameOverride
+fullnameOverride: ""
+
+# Overrides the deployment namespace in case of multi-namespace deployments, for example when using umbrella charts
+# where some sub-charts should be deployed in different namespaces.
+# This affects every resource deployed by this chart.
+# The default value is .Release.Namespace, if left unspecified.
+namespaceOverride: ""
+
+# Additional common labels to add to all resources for the application.
+# Values are processed as Helm templates.
+# Ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
+#
+# Example:
+# labels:
+#   foo: bar
+#   some-label: {{ .Values.someValue }}
+labels: {}
+
+# Additional common annotations to add to all resources for the application.
+# Values are processed as Helm templates.
+# Ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+#
+# Example:
+# annotations:
+#   foo: bar
+#   some-annotation: {{ .Values.someValue }}
+annotations: {}
+
+########################
+# Image Configurations #
+########################
+
+# Ref: https://kubernetes.io/docs/concepts/containers/images/
+image:
+  # The registry that hosts the application image.
+  # The default is to pull it from the Ontotext Registry, but it can be overridden to pull from other public or private registries.
+  registry: docker.io
+
+  # The repository name of the ESG API image.
+  repository: keycloak/keycloak-operator
+
+  # Image tag that corresponds to the version of application.
+  # By default, the chart uses .Chart.AppVersion to construct the full image name.
+  # Use this to override the value from .Chart.AppVersion and effectively deploy a custom version.
+  tag: "25.0.6"
```

Comment:
I think this should be empty and the value should come from the `appVersion` in the Chart.yaml.